migtests/scripts/run-config-test.sh

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ $# -gt 2 ]; then
+    echo "Usage: $0 TEST_NAME [env.sh]"
+    exit 1
+fi
+
+set -x
+
+export YB_VOYAGER_SEND_DIAGNOSTICS=false
+export TEST_NAME=$1
+
+export REPO_ROOT="${PWD}"
+export SCRIPTS="${REPO_ROOT}/migtests/scripts"
+export TESTS_DIR="${REPO_ROOT}/migtests/tests"
+export TEST_DIR="${TESTS_DIR}/${TEST_NAME}"
+export PYTHONPATH="${REPO_ROOT}/migtests/lib"
+
+CONFIG_TEMPLATE="${SCRIPTS}/config-templates/offline-migration.yaml"
+GENERATED_CONFIG="${TEST_DIR}/generated-config.yaml"
+CONFIG_GEN_SCRIPT="${SCRIPTS}/generate_config.py"
+
+# Order of env.sh import matters.
+if [ $2 != "" ] #if env.sh is passed as an argument, source it
+then
+    if [ ! -f "${TEST_DIR}/$2" ]
+	then
+		echo "$2 file not found in the test directory"
+		exit 1
+	fi
+	source ${TEST_DIR}/$2
+else
+	source ${TEST_DIR}/env.sh
+fi
+
+source ${SCRIPTS}/${SOURCE_DB_TYPE}/env.sh
+
+
+source ${SCRIPTS}/functions.sh
+
+normalize_and_export_vars "offline"
+
+source ${SCRIPTS}/yugabytedb/env.sh
+
+
+main() {
+	echo "Deleting the parent export-dir present in the test directory"
+	rm -rf ${EXPORT_DIR}	
+	echo "Creating export-dir in the parent test directory"
+	mkdir -p ${EXPORT_DIR}
+	echo "Assigning permissions to the export-dir to execute init-db, cleanup-db scripts"
+	chmod +x ${TEST_DIR}/init-db ${TEST_DIR}/cleanup-db
+
+	step "START: ${TEST_NAME}"
+	print_env
+
+    # Generate the config dynamically
```

Comment:
Indentation seems a little off at some places like this. Can you have a look at that?